linux_os/guide/system/auditing/policy_rules/audit_basic_configuration/rule.yml

```diff
@@ -73,4 +73,5 @@ template:
     vars:
         filepath: /etc/audit/rules.d/10-base-config.rules
         contents: |+
-            {{{ file_contents_audit_base_config|indent(12) }}}
+            {{{ file_contents_audit_base_config|indent(12) }}}#do not remove this comment
```

Comment:
Is it expected that this line contains a comment?
Does it need to be trailing the Jinja code?